gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -113,4 +126,39 @@ export class NotificationsListComponent implements OnInit {
       )
       .subscribe(() => this.ngOnInit());
   }
+
+  addNotification() {
+    this.matDialog
+      .open<NotificationsAddNotificationDialogComponent, NotificationsAddNotificationDialogData, NewNotification>(
+        NotificationsAddNotificationDialogComponent,
+        {
+          width: '500px',
```

Comment:
we have a UI-particules variable for dialog sizes. WDYT to use it ? 

https://particles.gravitee.io/?path=/story/components-dialog-readme--page

Many only Dialog not use it, this was introduced later ;)

Reply:
Done